backend/src/Designer/Controllers/PreviewController.cs

```diff
@@ -142,12 +142,6 @@ public async Task<ActionResult<ApplicationMetadata>> ApplicationMetadata(string
             string appNugetVersionString = _appDevelopmentService.GetAppLibVersion(AltinnRepoEditingContext.FromOrgRepoDeveloper(org, app, developer)).ToString();
             // This property is populated at runtime by the apps, so we need to mock it here
             applicationMetadata.AltinnNugetVersion = GetMockedAltinnNugetBuildFromVersion(appNugetVersionString);
-            if (altinnAppGitRepository.AppUsesLayoutSets())
-            {
-                LayoutSets layoutSets = await altinnAppGitRepository.GetLayoutSetsFile(cancellationToken);
-                applicationMetadata = SetMockDataTypeIfMissing(applicationMetadata, layoutSets);
```

Comment:
It looks like `SetMockDataTypeIfMissing` and `AddDataTypesToReturnedLayoutSetsIfMissing` can now be removed from `PreviewController`

Reply:
Good catch!